docs/sources/helm-charts/mimir-distributed/_index.md

```diff
@@ -8,15 +8,18 @@ keywords:
   - Grafana Enterprise Metrics
   - Grafana metrics
 cascade:
-  # Used to generate relative URL paths to related documentation
   mimir_docs_version: "v2.9.x"
   gem_docs_version: "v2.9.x"
 ---
 
 # Grafana mimir-distributed Helm chart documentation
 
-The mimir-distributed Helm chart for [Grafana Mimir](/docs/mimir/{{< param "mimir_docs_version" >}}/) and [Grafana Enterprise Metrics](/docs/enterprise-metrics/{{< param "gem_docs_version" >}}/) (GEM) allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
+The mimir-distributed Helm chart for [Grafana Mimir] and [Grafana Enterprise Metrics](/docs/enterprise-metrics/{{< param "gem_docs_version" >}}/) (GEM) allows you to configure, install, and upgrade Grafana Mimir or Grafana Enterprise Metrics within a Kubernetes cluster.
 
 > **Note:** By default, the mimir-distributed Helm chart documentation applies to both Grafana Mimir and GEM. If it only applies to GEM, it is explicitly stated.
 
 {{< section menuTitle="true" >}}
+
+{{% docs/reference %}}
+[Grafana Mimir]: "/ -> /docs/mimir/<MIMIR DOCS VERSION>"
```

Comment:
Let's rename to MIMIR_DOCS_VERSION for consistency, easy understanding and grep.